nixos/modules/system/boot/networkd.nix

```diff
@@ -303,6 +303,41 @@ let
 
       sectionTap = checkUnitConfig "Tap" tunChecks;
 
+      sectionL2TP = checkUnitConfig "L2TP" [
+        (assertOnlyFields [
+          "TunnelId"
+          "PeerTunnelId"
+          "Remote"
```

Comment:
I just noticed that `assertHasField` here also does not work. This is a bit unfortunate, since this section is a bit more complicated, but I guess this will also have to do without check.